crates/experimental-api/tests/rpc.rs

```diff
@@ -252,8 +252,7 @@ async fn test_getStorageAt() -> Result<(), Error> {
         "0x0341c1bdfd89f69748aa00b5742b03adbffd79b8e80cab5c50d91cd8c2a79be1",
     )?;
 
-    let block_id =
-        BlockId::BlockNumber { block_number: BlockNumber::try_new(600612)? };
```

Comment:
why this change is necessary?

Reply:
I have pushed the last commit with the revert back to 600612 for visualization. There is also a change in `ctx()` setup of node's `l1_state_root`. 

In `crates::experimental-api::src::proof_validation::mod.rs` there is a check `*state_commitment == parsed_global_root && global_root == parsed_global_root`. 

They didn't match because of the wrong `l1_state_root`, making the test fail and I have adapted `block_id` at first. Both tests are valid, with `Tag::Latest` and 600612, but when I was testing it on my local pc, I was constantly using `Tag::Latest` for calling rpc and it was easier for me to update the test with these values.